File: ViewModels/ViewModels/BinnacleVIewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ViewModels.ViewModels
{
    public class BinnacleViewModel : BaseViewModel
    {
        public DateTime Date { get; set; }

        public string Description { get; set; }

        public UserViewModel User { get; set; }
    }
}

File: src/musicgo/src/shared/classes/BinnacleViewModel.ts

import { UserViewModel } from './UserViewModel';
export class BinnacleViewModel{
    public Id: string = '00000000-0000-0000-0000-000000000000';
    public Description : string = "";
    public Date : Date = new Date();
    public User : UserViewModel | null = new UserViewModel();

}